### ISSUE TITLE
convenience starter

### ISSUE THREAD

**Issue by Dierk** [paf31/purescript-sdom#14]
the examples show some duplication in the main function, while it could just read:

`main = hangOn 'container' myComponent`

if we had a `hangOn` convenience function. This would also get rid of some immaterial imports.
One issue could be how best to handle error cases. Many fatal error cases are likely to be handled in some standard fashion. So providing one general handler (and allowing to replace it) should be fine.

**Comment by paf31** [comment]
Agreed, this would be useful. In other libraries, it's been called `defaultMain`.

**Comment by paf31** [comment]
One option is for the default function to use the document body as the container node. This removes the question of how to handle missing nodes.
